includes/notes/class-wc-payments-notes-instant-deposits-eligible.php

```diff
@@ -23,19 +23,18 @@ class WC_Payments_Notes_Instant_Deposits_Eligible {
 	/**
 	 * Name of the note for use in the database.
 	 */
-	// TODO: Get the proper doc url.
-	const NOTE_DOCUMENTATION_URL = '';
+	const NOTE_DOCUMENTATION_URL = 'https://docs.woocommerce.com/document/payments/';
```

Comment:
Maybe we could link deposits sections of docs? (https://docs.woocommerce.com/document/payments/#section-5) I think instant deposits info would be there. Not too important anyhow.

Reply:
The problem with linking like that is that the sections are dynamically named, so if a new section is added above, section 4 will become section 5. As of right now a new section needs to be added for instant deposits, so I made it just go to the page so the merchant could find their way from there.

Reply:
Good point! it works better when they use names for sections instead of numbers. Lets keep it then.